src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -251,7 +270,7 @@ export class HuiCardPicker extends LitElement {
         this._usedEntities
       );
 
-      if (!noElement) {
+      if (!noElement && !(customCard && !customCard.preview)) {
```

Comment:
```suggestion
      if (!noElement && (!customCard || customCard.preview)) {
```

Not sure if it is just me. But I saw this an had to think about the logic here. Maybe this is less confusing?

Reply:
DeMorgans theorem, eh? Haven't seen you in a while...